src/elements/module.rs

```diff
@@ -702,8 +702,7 @@ mod integration_tests {
 	}
 
 	#[test]
-	#[should_panic]
-	fn wrong_varuint1_case() {
+	fn varuint1_case() {
```

Comment:
Why this change was made?

Reply:
I was curious too! Apparently though this is a valid module with shared memory which fails validation earlier but passes afterwards